solarnet/peering.sh

```diff
@@ -1,9 +1,17 @@
 #!/usr/bin/env bash
 
-# Builds a cjdroute.conf snippet for peering with all nodes in secrets.yml
-#
-# Usage:
-#   ./peering.sh
+function usage {
+  echo "Builds a cjdroute.conf snippet for peering with the specified ansible hosts, using the specified password"
+  echo
+  echo "Usage:"
+  echo "  ./peering.sh <pattern> <password-name>"
+  echo "  ./peering.sh all solarnet"
+  echo "  ./peering.sh gateway alexandria"
```

Comment:
I think its just an example hes using for the usage text? Although the title of the PR makes me think it might be otherwise